src/utils/constants.js

```diff
@@ -6,3 +6,5 @@ export const TRASHED = 'TRASHED';
 export const BANNED = 'BANNED';
 export const ADMIN = 'ADMIN';
 export const AUTHOR = 'AUTHOR';
+export const SLUGONE = 'Hello-Article-5-3b8ab5fa-c594-4d5a-be6c-0b56888bb299';
```

Comment:
`SLUGONE` and `SLUGTWO` should not be in this file as they are used in tests, please put these constants in the test file.

Reply:
Alright. I will do that.

Reply:
I have done this already